packages/blade/src/components/Table/TableBody.web.tsx

```diff
@@ -263,10 +266,7 @@ const StyledRow = styled(Row)<{ $isSelectable: boolean }>(({ theme, $isSelectabl
           cursor: 'pointer',
         },
       }),
-      '&:focus': {
-        outline: 'none',
-        boxShadow: `0 0 0 2px ${getIn(theme.colors, 'brand.primary.300')}`,
-      },
+      '&:focus': getFocusRingStyles({ theme, negativeOffset: true }),
```

Comment:
What is negativeOffset for?

Reply:
Added comments on the getFocusRingStyles implementation. 

With negative offset: 
<img width="509" alt="image" src="https://github.com/razorpay/blade/assets/35374649/ddc24484-262b-43fd-b4ef-fca3e3bbaeba">


Without negative offset:
<img width="535" alt="image" src="https://github.com/razorpay/blade/assets/35374649/e0d31cd7-b3ee-472e-ad71-fc6edea0eb2b">